lib/va_profile/v2/contact_information/person_response.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'va_profile/response'
+require 'va_profile/models/person'
+# rubocop:disable ThreadSafety/InstanceVariableInClassMethod
```

Comment:
The rubocop inline disable is something we should avoid. The rule it's flagging could lead to a race condition (maybe not likely) but it's not thread safe here. I thin kit's complaining about response_body

Reply:
Yes, honestly, a lot of this code will be condensed. I just copied pasted it